components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -340,6 +344,20 @@ void BravePrefProvider::MigrateShieldsSettingsV1ToV2() {
   prefs_->SetInteger(kBraveShieldsSettingsVersion, 2);
 }
 
+void BravePrefProvider::MigrateDefaultHttpsContentSetting() {
+  auto* obsolete_pref = prefs_->Get(kObsoleteDefaultHttpsSetting);
```

Comment:
this is not correct, we should be migrating the default content setting and definitely not touching prefs directly like this

Reply:
done, thanks!